website/src/pages/pricing.tsx

```diff
@@ -32,33 +32,32 @@ export default ((props: any) => (
             </Helmet>
             <div className="pricing-page">
                 <ContentSection color="primary" className="hero-section text-center py-5">
-                    <h1>Sourcegraph pricing</h1>
-                    <h3>
-                        Free to start. No commitment.
-                        <br />
-                        Only pay for active users.
-                    </h3>
+                    <h2>Universal Code Search, for less than the cost of one bad line of code.</h2>
+                    <h3>Get started in minutes for free. Scales massively.</h3>
                 </ContentSection>
                 <div className="container-fluid pricing-page__plans">
                     <div className="row pt-6">
                         <div className="col-6 col-md-3 mx-auto mb-4">
                             <PricingPlan
                                 className="pricing__plan"
-                                name="Free"
+                                name="Starter"
                                 description="For individuals and small teams."
-                                price="Free"
+                                price="$0 (0-10 users), $10 (11-25 users)"
```

Comment:
Is the timeframe specified anywhere? If not, we should add "per year" and make it clear it is not per user but for the entire team < 25.

Reply:
I think this should be a one-time payment. Added to https://docs.google.com/document/d/1ybmvxonuhf4Y9pvU8BgcnAuu3emeHEdnqIBmg4XUj00/edit#, please comment there with your feedback.